tests/mesh/contains_point.C

```diff
@@ -51,11 +51,11 @@ public:
     elem->set_node(2) = &c;
 
     // midpoint
-    Point mid = 1.0/3.0 * (a + b + c);
+    Point mid = (a + b + c) / 3;
     CPPUNIT_ASSERT (elem->contains_point(mid));
```

Comment:
This should just be changed to one line of code, `elem->contains_point(elem->centroid())`.  I'll make that small change...